python/tank/templatekey.py

```diff
@@ -66,6 +76,27 @@ def __init__(self,
         if not all(self.validate(choice) for choice in self.choices):
             raise TankError(self._last_error)
     
+    @property
+    def choices(self):
+        """
+        :returns:    List of choices available for this key
+        """
+        return self._choices.keys()
+    
+    @property
+    def choice_labels(self):
```

Comment:
I was trying not to change the existing interface in a breaking way (so keeping choices) - I could instead add a method called something like 'get_choice_details()' that would return the full dictionary?